src/PhpSpreadsheet/Calculation/Calculation.php

```diff
@@ -228,7 +228,7 @@ class Calculation
     private static $phpSpreadsheetFunctions = [
         'ABS' => [
             'category' => Category::CATEGORY_MATH_AND_TRIG,
-            'functionCall' => [MathTrig::class, 'builtinABS'],
+            'functionCall' => [MathTrig\Abs::class, 'evaluate'],
```

Comment:
`absolute` might be a better method name in this case, no risk of confusion with the core PHP equivalent name

Reply:
Done.